### ISSUE TITLE
Camera view takes a long time to appear on M1 Mac

### ISSUE THREAD

**Issue by dsmurfin** [MetalPetal/MetalPetal#283]
The camera view in the sample app takes a very long time to load on an M1 Mac (up to 90 seconds).

### Checklist
- [x] I've read the [README](https://github.com/MetalPetal/MetalPetal/blob/master/README.md)
- [x] If possible, I've reproduced the issue using the `master` branch of this repo
- [x] I've searched for [existing GitHub issues](https://github.com/MetalPetal/MetalPetal/issues)

### Environment

 Info                    | Value                               |
-------------------------|-------------------------------------|
 MetalPetal Version      | 1.23.0
 Integration Method      | SwiftPM
 Platform & Version      | macOS 11.6
 Device                  | MacBook Pro 13'

### Steps to Reproduce
Open the example app and launch on an M1 Mac.

### Expected behavior
Launch would be consistent with Intel.

### Actual behavior
The camera view in the sample app takes a very long time to load on an M1 Mac, running macOS 11.6. I have been unable to try on a different version of macOS. The same issue presents in my application at launch which uses similar code to the CameraView. My app can take up to 90 seconds to launch. This issue is not present on an Intel Mac running macOS 11.6, or macOS 12. Running in Rosetta makes no difference.

It is possible the issue actually lies in VideoIO, but it is hard to tell.

**Comment by YuAo** [comment]
I don't have a M1 mac currently. 
Can you attach a call stack trace when the camera view is loading (using the "pause" debug button)?

**Comment by dsmurfin** [comment]
Here's the very strange thing. I've just fired up my M1 MacBook to get the stack trace for you and it's now working flawlessly in both the example app and in my app, when both were failing consistently when I raised this issue. Same version of macOS, same version of Xcode. Very strange!

I think we should close the issue for now and if I see it again I will get the call stack trace for you and send it over.

Thanks again for such a great library.
